NST/src/main/java/com/ebay/nst/NSTServiceWrapperProcessor.java

```diff
@@ -214,7 +214,7 @@ protected void schemaValidation(NSTServiceWrapper<? extends NSTSchemaValidator>
 		if ((!isSchemaValidationDisabled()) && (isSchemaValidate) && (schemaValidator != null)
 				&& !serviceWrapper.alwaysDisableSchemaValidation()) {
 			Reporter.log("Schema Validation Turned On.", true);
-			schemaValidator.toString(); // Print out the validator details including schema path and API choice.
+			Reporter.log(schemaValidator.toString()); // Print out the validator details including schema path and API choice.
```

Comment:
Why not log to std out too? 
```suggestion
			Reporter.log(schemaValidator.toString(), true); // Print out the validator details including schema path and API choice.
```

Reply:
Good catch. Missed that. Thank you.